docs/search/solr_search_engine.md

```diff
@@ -33,6 +33,10 @@ cp -R ./vendor/ibexa/solr/src/lib/Resources/config/solr/* server/ez/template
 cp server/solr/configsets/_default/conf/{solrconfig.xml,stopwords.txt,synonyms.txt} server/ez/template
 cp server/solr/solr.xml server/ez
 
+# If you are using Ibexa Commerce, additionally copy commerce-specific configuration files:
+cat ../vendor/ibexa/commerce-shop/src/bundle/Search/Resources/config/solr/custom-fields-types.xml >> server/ez/template/custom-fields-types.xml
```

Comment:
This example is a bit broken... in line 30 we have :
```
cd /opt/solr
```

so ibexa files are no longer in `./vendor` (line 32), or `../vendor` as specified here.. So maybe we should write something like this when refering to ibexa DXP files:
```suggestion
cat [IbexaDXP_installdir]/vendor/ibexa/commerce-shop/src/bundle/Search/Resources/config/solr/custom-fields-types.xml >> server/ez/template/custom-fields-types.xml
```